### ISSUE TITLE
Walberla: LBFluidWalberla node access

### ISSUE THREAD

**Issue by reinaual** [espressomd/espresso#4422]
When trying to access the node properties of the LBFluid without adding it as an actor first results in an error although a user would expect it to work.
```python
import espressomd
import espressomd.lb

system = espressomd.System(box_l=[10, 10, 10])
system.time_step = 1.0

lattice = espressomd.lb.LatticeWalberla(n_ghost_layers=1, agrid=1.0)

lbf = espressomd.lb.LBFluidWalberla(lattice=lattice, viscosity=1.0, density=1.0, tau=1.0, agrid=1.0)

lbf[1, :, :].density = 1.0
```

```
Traceback (most recent call last):
  File "../script/lb_issue.py", line 11, in <module>
    lbf[1, :, :].density = 1.0
  File "/ssd/areinauer/espresso/rudolf/build/src/python/espressomd/lb.py", line 97, in __getitem__
    return LBSlice(key, self.shape)
  File "/ssd/areinauer/espresso/rudolf/build/src/python/espressomd/lb.py", line 779, in __init__
    self.x_indices, self.y_indices, self.z_indices = self.get_indices(
  File "/ssd/areinauer/espresso/rudolf/build/src/python/espressomd/lb.py", line 783, in get_indices
    x_indices = np.atleast_1d(np.arange(shape_x)[key[0]])
IndexError: index 1 is out of bounds for axis 0 with size 0
```

**Comment by jngrad** [comment]
This is something that we'll have to deal with eventually. Right now the LB class is an actor, and as such inherits the two-step constructor logic from all actors. This design decision is only useful to actors that change globals (e.g. P3M), because we need to be able to alter actor parameters even when the actor hasn't been activated yet. That's why we can change the density of the LB actor but not of its nodes, until the actor is activated for the first time. Over the last 2 months the core has been relying less and less on the LB shared pointer global, at some point the observables and integrators will be the last consumers of the LB global and it should be possible to remove this two-step logic from the LB actor.

**Comment by jngrad** [comment]
This should be fixed by 5bdb614fbe595b67. The script interface was fully decoupled from the global shared pointer by ebf8a9f0bd3d6d8.
